cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -137,6 +137,7 @@ def test_vmware_vds_ui_display(soft_assert, appliance, provider):
 
 
 @pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10", "5.11")
```

Comment:
I think we should just `skip` with a small explanation as to why we are skipping.

Reply:
Actually I did this as @mshriver suggested in his email to do so.